index.ios.js

```diff
@@ -1,27 +1,4 @@
-import React, { Component } from 'react';
-import {
-  AppRegistry,
-  StatusBar,
-  View,
-} from 'react-native';
-import { theme } from './config';
+import { AppRegistry } from 'react-native';
```

Comment:
I tried that but it seems that these filenames are hardcoded and expected by React Native.